packages/analytics-js/src/components/utilities/loadOptions.ts

```diff
@@ -107,7 +109,47 @@ const normalizeLoadOptions = (
   return mergedLoadOptions;
 };
 
-const getSourceConfigURL = (configUrlHost = DEFAULT_CONFIG_BE_URL): string =>
-  `${configUrlHost}/sourceConfig/?p=${MODULE_TYPE}&v=${APP_VERSION}&build=${BUILD_TYPE}`;
+const getSourceConfigURL = (
+  configUrl: string,
+  writeKey: string,
+  lockIntegrationsVersion: boolean,
+): string => {
+  const defSearchParams = new URLSearchParams({
```

Comment:
I think this will break in IE11. Have you tested?

URLSearchParams need polyfill and these are loaded after we generate the url and get response back I think

Reply:
Yes, it works in IE11. Loading configuration occurs post polyfill load in the life cycle.